pkg/apis/serving/v1alpha2/framework_tensorflow_test.go

```diff
@@ -110,6 +110,7 @@ func TestTensorflowContainer(t *testing.T) {
 			"--rest_api_port=" + TensorflowServingRestPort,
 			"--model_name=someName",
 			"--model_base_path=/mnt/models",
+			"--rest_api_timeout_in_ms=0",
```

Comment:
This should get the default value from configmap right?

Reply:
thanks for keeping me honest with the tests, fixed in 2c3b5de